src/styles/main.scss

```diff
@@ -1,3 +1,79 @@
+$month-length: 31;
+$calendar-padding: 10px;
+$box-width: 100px;
+$box-height: 100px;
+$box-gap: 1px;
+$font-size: 30px;
+$days: (0: 'mon', 1: 'tue', 2: 'wed', 3: 'thu', 4: 'fri', 5: 'sat', 6: 'sun');
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
   margin: 0;
+  
+  height: 100vh;
+}
+
+.calendar {
```

Comment:
Don't put all in one files, create a separate file for each BEM block

Reply:
fixed